### ISSUE TITLE
My Grave was destroyed by another mod. Is there any way to get it back?

### ISSUE THREAD

**Issue by ChemCaps** [B1n-ry/Youre-in-grave-danger#170]
I died while playing on a Bisecthosting server with some friends and one of them accidentally destroyed my grave by summoning a boss from another mod. I had a lot of gear that I don't exactly just want to go and duplicate, especially since some of it was in backpacks and I can't see what was in those. Is there a way to restore my inventory from my last death or do I just need to guess from what I had?

**Comment by B1n-ry** [comment]
Yes you can restore graves if you're a server operator. If it's your latest death you can run the command `/yigd restore [playername]` to restore their latest grave.
You could also (I recommend this way as it's clearer which grave you're restoring) run the command `/yigd grave [playername]`. This will bring up a GUI containing all the player's graves. By filtering on destroyed graves, you should be able to find the one that was destroyed. Clicking on that will bring up another GUI where all the items lost can be viewed, and if you're a server operator there will be a few buttons on the side. One of them (when you hover over it) will say "restore". This will restore the grave to the player that died.

What boss destroyed the grave btw? I'd be happy to try and stop that from happening again by implementing some sort of compatibility

**Comment by B1n-ry** [comment]
I'm unsure which boss destroyed the grave, but I think that the issue should be resolved, and that the grave should no longer be destructible by this boss (and hopefully many other things that might have broken graves before)

**Comment by Holgast** [comment]
hello, I'm using a modpack (Craftonia) that is using the 2.0.9 version of this mod and the Netherite Monstrosity from Cataclysm destroyed my grave. just wanted to ask whether that particular one has been fixed in newer versions or bring it to your attention if not
